content/docs/other-guides/accessing-uis.md

```diff
@@ -36,6 +28,15 @@ The central UI dashboard looks like this:
   alt="Kubeflow central UI"
   class="mt-3 mb-3 border border-info rounded">
 
+## Overview of accessing the Kubeflow UIs
+
+To access the Kubeflow UIs you need to connect to the 
```

Comment:
Suggestion: add a comma after UIs

To access the Kubeflow UIs, you need to connect to the

Reply:
Done